### PR TITLE
TRD digit params class

### PR DESCRIPTION
**Pull request by jolopezl**
Helper class for TRDDigitManager.

### CODE REVIEW

File: Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    }
+  }
+  void setPretriggerPhaseAll(unsigned int phase)
+  {
+    for (auto& el : mPretriggerPhase)
+      el = phase;
+  }
+  void setADCbaselineAll(int baseline)
+  {
+    for (auto& el : mADCbaseline)
```
Comment (sawenzel) [inline Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h]: please add braces {}

File: Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+  void setNTimeBinsAll(int ntb)
+  {
+    for (auto& el : mNTimeBins) {
+      el = ntb;
+    }
+  }
+  void setPretriggerPhaseAll(unsigned int phase)
+  {
+    for (auto& el : mPretriggerPhase)
```
Comment (sawenzel) [inline Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h]: braces

Comment (jolopezl) [reply Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h]: thanks, included

File: Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h (hunk 1 above)
Comment (jolopezl) [reply Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h]: thanks, included

File: Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+class TRDDigitsParam
+{
+
+ public:
+  TRDDigitsParam() = default;
+  TRDDigitsParam(const TRDDigitsParam& p);
+  TRDDigitsParam& operator=(const TRDDigitsParam& p);
+  ~TRDDigitsParam() = default;
+
+  void Copy(TRDDigitsParam& p) const;
```
Comment (sawenzel) [inline Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h]: I wondering if we need copy constructor and custom copy functions? In particular the Copy function is not actually doing some non-trivial. (So the default compiler generated functions might be good enough?)

Comment (jolopezl) [reply Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h]: Hello @sawenzel, I agree with you. I have removed this unnecessary methods.

File: Detectors/TRD/base/src/TRDDigitsParam.cxx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ * about the suitability of this software for any purpose. It is          *
+ * provided "as is" without express or implied warranty.                  *
+ **************************************************************************/
+
+/* $Id: TRDDigitsParam.cxx 34070 2009-08-04 15:34:53Z cblume $ */
+
+///////////////////////////////////////////////////////////////////////////////
+//                                                                           //
+//  Class containing parameters for digits                                   //
+//  2019 - Ported from AliRoot to O2 (Jorge Lopez)                           //
```
Comment (sawenzel) [inline Detectors/TRD/base/src/TRDDigitsParam.cxx]: Seems like this file is no longer needed?

File: Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+namespace o2
+{
+namespace trd
+{
+class TRDDigitsParam
+{
+
+ public:
+  TRDDigitsParam() = default;
+  ~TRDDigitsParam() = default;
```
Comment (sawenzel) [inline Detectors/TRD/base/include/TRDBase/TRDDigitsParam.h]: thanks for cleaning up!

File: Detectors/TRD/base/src/TRDDigitsParam.cxx (hunk 4 above)
Comment (jolopezl) [reply Detectors/TRD/base/src/TRDDigitsParam.cxx]: All right, removing this file then. I just thought we had the keep one cxx file per h file.

**Pull request merged**